package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "rdflib",
   "description": "an RDF library for node.js. Suitable for client and server side.",
-  "version": "0.8.0",
+  "version": "0.8.1",
```

Comment:
Ignore this version bump - it was from a previous npm publish, forgot to push to master.